server/src/main/java/org/opensearch/search/aggregations/support/MissingValues.java

```diff
@@ -359,7 +359,7 @@ public long getValueCount() {
 
             @Override
             public int docValueCount() {
-                return values.docValueCount();
+                return Math.max(1, values.docValueCount());
```

Comment:
I prefer to fix in `MissingValues`, not in `GlobalOrdinalsStringTermsAggregator`, `MissingValues` ​​may also be called from other places.